lib/ui/auth/login_page/login_viewmodel.dart

```diff
@@ -48,13 +49,11 @@ class LoginViewModel extends BaseViewModel {
   }
 
   Future<void> checkLoginStatus() async {
-    // try {
-    //   await _auth.checkToken();
-    //   _goToHome();
-
-    // } catch (e) {
-    await Future.delayed(Duration(milliseconds: 1000));
-    _navigationService.navigateTo(Routes.loginView);
-    // }
+    try {
+      await _user.fetchAndSetUserData();
+      _navigationService.navigateTo(Routes.workspaceView);
+    } catch (e) {
+      _navigationService.navigateTo(Routes.loginView);
```

Comment:
We don't need to navigate again, It's better we have a toast that tells the use couldn't login.

Reply:
Solved.